src/main.py

```diff
@@ -233,26 +227,47 @@ def internet_on():
 		return False
 
 
-def gettoken():
-	token = mc.get("access_token")
-	refresh = config['alexa']['refresh_token']
-	if token:
-		return token
-	elif refresh:
+class Token(object):
+
+	_token = None
+	_timestamp = None
+	_validity = 3570
+
+	def __init__(self, aconfig):
+
+		self._aconfig = aconfig
+
+		self.renew()
+
+	def get(self):
```

Comment:
Just as idea: rename this method to `token` and decorate it with `@property', then call to this will look like: 

```
headers = {'Authorization': 'Bearer %s' % token.token}
```
or even changing that method to `__str__`, so call above will look like this:
```
headers = {'Authorization': 'Bearer %s' % str(token)}
```